src/features/game/events/landExpansion/moveCrop.ts

```diff
@@ -126,6 +129,70 @@ export function isLocked(
     }
   }
 
+  if (buildings["Basic Composter"]?.[0]) {
+    const basicComposter = buildings["Basic Composter"][0];
+
+    const isComposting =
+      basicComposter.producing &&
+      basicComposter.producing?.readyAt > Date.now();
+
+    const composterCoordinates = basicComposter.coordinates;
+    const composterDimensions = BUILDINGS_DIMENSIONS["Basic Composter"];
+
+    const scarecrowPosition: Position = {
```

Comment:
The danger of copy/paste -- a million references to `scarecrow` that aren't related to scarecrow at all.